subprojects/docs/src/docs/userguide/publishing_overview.adoc

```diff
@@ -13,78 +13,76 @@
 // limitations under the License.
 
 [[publishing_overview]]
-== Publishing
+= Publishing
 
 The vast majority of software projects build something that aims to be consumed in some way. It could be a library that other software projects use or it could be an application for end users. _Publishing_ is the process by which the thing being built is made available to consumers.
 
 In Gradle, that process looks like this:
 
- 1. Define <<publishing_overview:what,what>> to publish
- 2. Define <<publishing_overview:where,where>> to publish it to
- 3. <<publishing_overview:how,Do>> the publishing
+ 1. Define <<#publishing_overview:what,what>> to publish
+ 2. Define <<#publishing_overview:where,where>> to publish it to
+ 3. <<#publishing_overview:how,Do>> the publishing
 
 Each of the these steps is dependent on the type of repository to which you want to publish artifacts. The two most common types are Maven-compatible and Ivy-compatible repositories, or Maven and Ivy repositories for short.
 
-NOTE: Looking for information on upload tasks and the `archives` configuration? See the <<artifact_management>> chapter.
+NOTE: Looking for information on upload tasks and the `archives` configuration? See the <<artifact_management.adoc#artifact_management,Artifact Management>> chapter.
```

Comment:
❌ This link reads "Legacy publishing" before (like it's heading) and now is "Artifact Management".